apps/api/src/dashboard/dashboard.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { Applicant } from '@/entity/applicant.entity';
+import { Ticle } from '@/entity/ticle.entity';
+
+@Injectable()
+export class DashboardService {
+  @InjectRepository(Ticle)
+  private ticleRepository: Repository<Ticle>;
+  @InjectRepository(Applicant)
+  private applicantRepository: Repository<Applicant>;
+
```

Comment:
```suggestion
  constructor(  
    @InjectRepository(Ticle)
    private ticleRepository: Repository<Ticle>,
    @InjectRepository(Applicant)
    private applicantRepository: Repository<Applicant>
  ) {}

```

다른분들과 동일하게 `constructor` 내부에서 private property를 정의하도록 수정해주세요!